docs/docs/model.md

```
@@ -439,6 +439,97 @@
 }
 ```
 
+### LabelledAs <Badge text="7.85.0+"/>
```

Markdownlint (0.35.0)

Element: Badge
Inline HTML

(MD033, no-inline-html)